.evergreen/run-tests.sh

```diff
@@ -119,20 +119,22 @@ if [ -n "$TEST_ENCRYPTION" ]; then
     # Use the nocrypto build to avoid dependency issues with older windows/python versions.
     BASE=$(pwd)/libmongocrypt/nocrypto
     if [ -f "${BASE}/lib/libmongocrypt.so" ]; then
-        export PYMONGOCRYPT_LIB=${BASE}/lib/libmongocrypt.so
```

Comment:
This seems weird. Why is it getting rid of the export in some branches but not others.

Reply:
Only one of them had been flagged, I had meant to standardize it, but I missed this one.  Updated.